docs/environment-variables.md

```diff
@@ -41,6 +41,14 @@
 | ENABLE_COMPONENTS_LIBRARY           | Whether to enable the component library and allow it to be browsed on this environment                                                                                                                                           | false                                                          |
 | ENABLE_DEVELOPMENT_TOOLS            | Whether to enable the development tools and allow it to be browsed on this environment                                                                                                                                           | false                                                          |
 
+## External Services
```

Comment:
Wonder whether this needs a separate heading? You could say that the API and the Auth service are also "external services". Wonder if it's easier just to leave them in the main table for consistency.

Reply:
tbh I was wondering about breaking out the other API keys into their own sections too. Having `LOCATIONS_BATCH_SIZE`, `SESSION_NAME`, `API_SECRET` and `ORDNANCE_MAP_API_URL` in the same section really did seem like a very very general grouping (which I know is the heading)

Reply:
I'll make that reordering a separate cleanup PR :)